Mathlib/CategoryTheory/Triangulated/Functor.lean

```diff
@@ -116,6 +116,25 @@ lemma map_distinguished [F.IsTriangulated] (T : Triangle C) (hT : T ∈ distTria
     F.mapTriangle.obj T ∈ distTriang D :=
   IsTriangulated.map_distinguished _ hT
 
+namespace IsTriangulated
+
+open ZeroObject
+
+variable [F.IsTriangulated]
+
+instance : PreservesZeroMorphisms F where
```

Comment:
Should this priority be lowered?

Reply:
Thanks for the suggestion. I am lowering the priority of these instances.